pkg/models/query.go

```diff
@@ -132,13 +132,18 @@ func (tabularConditions *TwinMakerTabularConditions) ToTwinMakerTabularCondition
 	return tabularCondition
 }
 
+type TwinmakerPropertyInfo struct {
+	PropertyName        string `json:"propertyName,omitempty"`
+	PropertyDisplayName string `json:"propertyDisplayName,omitempty"`
+}
+
 // TwinMakerQuery model
 type TwinMakerQuery struct {
 	GrafanaLiveEnabled bool                          `json:"grafanaLiveEnabled,omitempty"`
 	IsStreaming        bool                          `json:"isStreaming,omitempty"`
 	WorkspaceId        string                        `json:"workspaceId,omitempty"`
 	EntityId           string                        `json:"entityId,omitempty"`
-	Properties         []*string                     `json:"properties,omitempty"`
+	Properties         []*TwinmakerPropertyInfo      `json:"properties,omitempty"`
```

Comment:
This change seems like it would break any existing saved dashboards 🤔

Reply:
nice catch, any suggestion as to how I should handle it.